### ISSUE TITLE
Limit or change the projection on FetchPlans ?

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#4323]
OrientDb 2.0.10
Studio and Oriento

Simple case:

```
select reference as ref from V fetchPlan ref:1
```

The fetched records has all available fields and no projection. This leads to a messy dataset and big effort on the server side to map all these data.

**Comment by lvca** [comment]
Sorry, could you elaborate more the problem?

**Comment by StarpTech** [comment]
When I want to fetch my tree with embedded records I can configure this by the fetchPlan setting but what happens when I don't want to include the whole embedded record but only 4 properties or these 4 properties with a different projection? In my current situation I couldn't find any solution unless to map all my data serverside. It is clear now?

**Comment by StarpTech** [comment]
Status?

**Comment by StarpTech** [comment]
@lvca I think this is a common scenario and should be added as enhancement.

**Comment by red-0ne** [comment]
Have you tried the `include`/`exclude` methods ?
```sql
SELECT *, reference.include('include', 'these', 'fields') as ref FROM V FETCHPLAN ref:1
```

**Comment by lvca** [comment]
@StarpTech Did you solve with include/exclude?

**Comment by pmig** [comment]
In order to achieve this you don't even need a fetchplan:

**Get connected edge attributes:**
```
SELECT attr, out_ref.include('edgeattr1','edgeattr2') as attrs FROM class;
```

**Get connected vertex attributes:**
```
SELECT attr, out_ref.in.include('edgeattr1','edgeattr2') as attrs FROM class;
```
OR (_depends on your edge orientation_)
```
SELECT attr, out_ref.out.include('edgeattr1','edgeattr2') as attrs FROM class;
```

@lvca I think that this issue can be closed and is pretty similar to: #5173 and #1938
